ui/src/pages/namespace/Events/History/EventsList.tsx

```diff
@@ -109,29 +110,32 @@ const EventsList = ({
                   )}
                 </EventsTable>
               </Card>
-              {totalPages > 1 && (
-                <Pagination>
-                  <PaginationLink
-                    data-testid="pagination-btn-left"
-                    icon="left"
-                    onClick={() => goToPreviousPage()}
-                  />
-                  {pagesList.map((page) => (
+              <div className="flex items-center justify-end gap-2">
+                <SelectPageSize goToPage={() => goToPage(1)}></SelectPageSize>
```

Comment:
If we don't pass any children, we might use `<SelectPageSize goToPage={() => goToPage(1)} />`?